src/Documenter.jl

```diff
@@ -23,11 +23,14 @@ import Unicode
 import Pkg
 import RegistryInstances
 import Git
+import TimerOutputs
 # Additional imported names
 using Test: @testset, @test
 using DocStringExtensions: SIGNATURES, EXPORTS
 using Base64: base64decode
 
+const TIMER = Ref(TimerOutputs.TimerOutput())
```

Comment:
I think this should go into the `Document` context object that gets passed everywhere. Don't really want to deal with global state like this.

Reply:
refactored